validation/security-configs_test.go

```diff
@@ -61,36 +38,29 @@ func testSecurityConfigOfChain(t *testing.T, chainID uint64) {
 	// Portal version `3` is the first version of the `OptimismPortal` that supported the fault proof system.
 	isFPAC := majorVersion >= 3
 
-	if isFPAC {
-		contractCallResolutions = append(contractCallResolutions,
-			resolution{"DisputeGameFactoryProxy", "admin()", "ProxyAdmin"},
-			resolution{"DisputeGameFactoryProxy", "owner()", "ProxyAdminOwner"},
-			resolution{"AnchorStateRegistryProxy", "admin()", "ProxyAdmin"},
-			resolution{"DelayedWETHProxy", "admin()", "ProxyAdmin"},
-			resolution{"OptimismPortalProxy", "guardian()", "Guardian"},
-			resolution{"OptimismPortalProxy", "systemConfig()", "SystemConfigProxy"},
-		)
-	} else {
-		contractCallResolutions = append(contractCallResolutions,
-			resolution{"OptimismPortalProxy", "GUARDIAN()", "Guardian"},
-			resolution{"OptimismPortalProxy", "SYSTEM_CONFIG()", "SystemConfigProxy"},
-			resolution{"OptimismPortalProxy", "L2_ORACLE()", "L2OutputOracleProxy"},
-			resolution{"L2OutputOracleProxy", "admin()", "ProxyAdmin"},
-			resolution{"L2OutputOracleProxy", "CHALLENGER()", "Challenger"},
-		)
-	}
+	contractCallResolutions := standard.Config[OPChains[chainID].Superchain].L1.GetResolutions(isFPAC)
 
-	for _, r := range contractCallResolutions {
-		contractAddress, err := Addresses[chainID].AddressFor(r.name)
-		require.NoError(t, err)
+	for contract, methodToOutput := range contractCallResolutions {
 
-		want, err := Addresses[chainID].AddressFor(r.shouldResolveToAddressOf)
+		contractAddress, err := Addresses[chainID].AddressFor(contract)
 		require.NoError(t, err)
 
-		got, err := getAddressWithRetries(r.method, contractAddress, client)
-		require.NoErrorf(t, err, "problem calling %s.%s", contractAddress, r.method)
+		for method, output := range methodToOutput {
+
+			var want Address
+			if strings.HasPrefix(output, "0x") {
+				want = MustHexToAddress(output)
```

Comment:
This is in anticipation of some declarations I am about to make which will me raw contract addresses (i.e. superchain-wide multisig addresses) instead of a contract name.